source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -1,16 +1,18 @@
 using System;
+using System.Collections.Concurrent;
```

Comment:
This is copied from Tentacle. The previous version did not play nicely with Visual Studio